### PR TITLE
refactor(SelectFilter): FRMW-411-covert to quark.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-411

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxSelectFilter.SelectFilter` to Quark service.

### CODE REVIEW

File: src/components/rxSelectFilter/rxSelectFilter.module.js [hunk 1]
```diff
@@ -12,11 +12,9 @@
  *
  * ## Filters
  * * {@link rxSelectFilter.filter:Apply Apply}
- *
- * ## Services
- * * {@link rxSelectFilter.service:SelectFilter SelectFilter}
  */
 angular.module('encore.ui.rxSelectFilter', [
+	'encore.ui.quarks',
```
Comment (jgonz0010) [inline src/components/rxSelectFilter/rxSelectFilter.module.js]: Tabs instead of spaces.

**Comment by 100stacks** [comment]
@jgonz0010, tab converted to spaces :+1:

**Comment by 100stacks** [comment]
## `SelectFilter` Quark Service

### Demo Page
screen shot 2015-12-29 at 3 05 41 pm

### API Docs
screen shot 2015-12-29 at 3 04 45 pm

**Comment by jgonz0010** [comment]
Looks Good to Me.

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/quarks.module.js [hunk 2]
```diff
@@ -21,5 +21,6 @@
  * * {@link quarks.service:hotkeys hotkeys}
  * * {@link quarks.service:SessionStorage SessionStorage}
  * * {@link quarks.service:rxBreadcrumbsSvc rxBreadcrumbsSvc}
- */
+ * * {@link quarks.service:SelectFilter SelectFilter}
+  */
```
Comment (CITguy) [inline src/quarks/quarks.module.js]: _nit picking_: remove leading space to line up the asterisk

Comment (CITguy) [inline src/quarks/quarks.module.js]: _my OCD kicked in_ :)

**Comment by CITguy** [comment]
*Note:* this needs to be merged before #1463

File: src/quarks/quarks.module.js (hunk 2 above)
Comment (100stacks) [inline src/quarks/quarks.module.js]: @CITguy that's a good thing :wink:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**